lib/screens/movie_home_page.dart

```diff
@@ -1,18 +1,21 @@
 import 'package:flutter/material.dart';
-import '/custom_widgets/movie_body.dart';
-import '/custom_widgets/movie_app_bar.dart';
+import '/movie_widgets/movie_app_bar.dart';
+import '/movie_widgets/movie_body.dart';
+
 import '/data_classes/movie.dart';
 
 class MovieHomePage extends StatelessWidget {
-  MovieHomePage({super.key});
+  const MovieHomePage({super.key});
 
-  final Movie movie = Movie.defaultMovie();
+  static const String routeName = "/movie";
 
   @override
   Widget build(BuildContext context) {
+    final Movie? movieFromNamed =
+        ModalRoute.of(context)?.settings.arguments as Movie?;
```

Comment:
Don't declare this as nullable if you'll force-unwrap it later. Either cast as `Movie!` here or check that if it's null show an error message.
Personally, I'd take the second approach :)

Reply:
fixed